app/common/containers/forms/DateFilterForm/index.js

```diff
@@ -14,14 +14,14 @@ import styles from './styles.scss';
 })
 export default class DateFilterForm extends React.Component {
   render() {
-    const { handleSubmit, submitting } = this.props;
+    const { handleSubmit, submitting, items } = this.props;
```

Comment:
It will be more readable if we'll destructure items right here:

```js
const { handleSubmit, submitting, items: [startFieldName, endFieldName] } = this.props;
```